src/io/flutter/logging/FlutterLogView.java

```diff
@@ -663,7 +631,7 @@ protected void onScrollToEnd() {
     dataPane = ScrollPaneFactory.createScrollPane(
       dataPanel,
       ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED,
-      ScrollPaneConstants.HORIZONTAL_SCROLLBAR_AS_NEEDED);
+      ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
```

Comment:
is it possible to keep the HORIZONTAL_SCROLLBAR_AS_NEEDED and still have the UI lay out as expected in IntelliJ? If not it is fine. Could be nice if users were able to scroll horizontally to see lines that intentionally do not wrap.

Reply:
I couldn't get that to work (though I tried). I'll add it to my list of things to revisit.